provider/provider.go

```diff
@@ -72,13 +75,14 @@ func Call(provider, specPath string) (string, error) {
 
 // Given a naked filename, returns a path to executable prefixed with DefaultPath
 // This is so that "./provider" will work as expected.
```

Comment:
This is outside of the scope of your change, but this comment looks like it needs updating... or I'm confused. The first line talks about handling a naked filename, then the second line talks about relative paths.